tests/Feature/WebAuthTest.php

```diff
@@ -30,6 +30,7 @@ public function setUp(): void
 	{
 		parent::setUp();
 		$this->setUpRequiresEmptyWebAuthnCredentials();
+		config(['app.url' => 'https://localhost']);
```

Comment:
This allows to just run tests locally without having to think whether the `.env` matches on not.